envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdater.kt

```diff
@@ -87,28 +91,40 @@ class SnapshotUpdater(
         // see GroupChangeWatcher
         return onGroupAdded
             .publishOn(globalSnapshotScheduler)
-            .measureBuffer("snapshot-updater-groups-published", meterRegistry)
+            .measureBuffer("snapshot-updater.count.total", meterRegistry)
             .checkpoint("snapshot-updater-groups-published")
-            .name("snapshot-updater-groups-published").metrics()
+            .name("snapshot-updater.count.total")
```

Comment:
snapshot_updater/snapshot.updater - . is mapped to _, right?

Reply:
. is, - isn't